src/pages/contributing/get-started.mdx

```diff
@@ -0,0 +1,164 @@
+smoothly and utilize our markdown templates on the
+[documentation](contributing/documentation/) page. A list of components that
+still need documentation help can be found
+[here](https://github.com/carbon-design-system/carbon-website/issues/2085).
+
+### Fixing issues and bugs
+
+If you find a bug in the source code, you can help us by
+[filling out a bug report](https://github.com/carbon-design-system/carbon/issues/new/choose)
+and, ideally, submitting a pull request with a fix.
+
+### Icons and pictograms
+
+You can find instructions on how to create and contribute
+[UI icons](https://www.ibm.com/design/language/iconography/ui-icons/contribute),
+[app icons](https://www.ibm.com/design/language/iconography/app-icons/production),
+and
+[pictograms](https://www.ibm.com/design/language/iconography/pictograms/contribute)
+on the IBM Design Language site.
+
+## The process
+
+Whether you're sharing a proposal or contributing code, designs, and
+documentation, there's a process to follow. Here's what it looks like:
+
+### Volunteer for existing work
+
+One way to volunteer is to look through
+[existing GitHub issues](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22)
+labeled with `community contribution`. You can also filter by role:
+[developer](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22++label%3A%22role%3A+dev+%F0%9F%A4%96%22_)
+or
+[designer](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22++label%3A%22role%3A+design+%3Apencil2%3A%22+).
+Put a comment in the issue saying you'd like the help.
+
+The Carbon team is also happy to help you find a good first issue. Just stop by
+our [office hours](https://carbondesignsystem.com/whats-happening/meetups/) or
+send us a slack message.
+
+### Submit an idea
+
+Have a new idea that you think would benefit Carbon? Or do you need to report a
+bug? First, be sure to look through the
+[issue backlog](https://github.com/carbon-design-system/carbon/issues) to make
+sure it is a novel idea or bug. Then, file your proposal on GitHub using the
+[issue templates](https://github.com/carbon-design-system/carbon/issues/new/choose).
+If you're willing to work on this idea yourself, be sure to let is know in your
+issue! Your idea will then go through a triage process by the Carbon team.
+
+### Iterate and get feedback
+
+Most contributors work in groups of 2-3 and either set up weekly sessions or
+join [meetups](https://carbondesignsystem.com/whats-happening/meetups/) such as
+the Figma Guild, Data Viz Guild, or office hours. In these sessions, it is
+common to share work in progress and ask lots of questions. As you make
+progress, update your GitHub issue.
+
+### Review and published
+
+Most work that results in code will be reviewed directly in a pull request.
```

Comment:
```suggestion
Work that results in code will be reviewed directly in a pull request.
```
Not sure we need "most", could also be "All work." or "Any work.."

Reply:
Thanks! I changed it to 'Work'